pkg/queue/manager.go

```diff
@@ -178,6 +178,7 @@ func (m *Manager) AddQueue(ctx context.Context, q *kueue.Queue) error {
 	if cq != nil && cq.AddFromQueue(qImpl) {
 		m.cond.Broadcast()
 	}
+	qImpl.reportPendingWorkloads()
```

Comment:
we will eventually get events for the workloads, doesn't look like this is necessary.

Reply:
what if the events for all the workloads came in before the queue?

Reply:
sg